FlutterFire.md

```diff
@@ -0,0 +1,34 @@
+# FlutterFire
+
+FlutterFire is a set of [Flutter plugins](https://flutter.io/platform-plugins/)
+that enable Flutter Apps to use one or more [Firebase](https://firebase.google.com/) services.
+
+## Supported Firebase Products
```

Comment:
Just want to be sure you want to use the word "Supported" here. We do support them, but Firebase team doesn't support them.